### PR TITLE
Add Leachate

### PR DESCRIPTION
**Pull request by abdusabri**
**Definitions**
- **Google:** water that has percolated through a solid and leached out some of the constituents
- **[Wikipedia](https://en.wikipedia.org/wiki/Leachate):** A leachate is any liquid that, in the course of passing through matter, extracts soluble or suspended solids, or any other component of the material through which it has passed

### CODE REVIEW

File: packages/scientific_terms_US/custom_scientific_US.dic.txt [hunk 1]
```diff
@@ -320904,6 +320904,7 @@ LEDs
 Le
 Leach
 Leach's
+Leachate
```
Comment (Jason3S) [inline packages/scientific_terms_US/custom_scientific_US.dic.txt]: Thank you. Please use `leachate`. `Leach` is a persons name.

**Comment by Jason3S** [comment]
Thank you very much. With up coming versions of cspell, the dictionaries will be case sensitive. 

`Leachate` would be a proper name and would only match against `Leachate`
`leachate` would be a term and will match against `leachate`, `Leachate`, and `LEACHATE`

**Comment by Jason3S** [comment]
Thank you

**Comment by abdusabri** [comment]
> Thank you

You are most welcome! 🙂

**Pull request merged**